torch/csrc/jit/passes/quantization.cpp

```diff
@@ -147,6 +148,10 @@ class InsertObserversHelper {
   void insertObservers(script::Module& module, const std::string& method_name);
 
  private:
+  ModuleMethodVector getInvokedMethods(
```

Comment:
The signature would be simpler if we just returned `std::vector<Method*>` or `std::vector<Graph*>` (`insertObservers` can also accept method/graph instead of module-method_name pair). Could you please change that?